cage/cryptids/static/cryptids/animals.css

```diff
@@ -0,0 +1,128 @@
+.image2 {
+    display: flex;
+    flex-direction: row;
+    height: 80px;
+    margin: 8px;
+    width: 80px;
+    border-radius: 55%;
+    border: 8px solid #23a897;}
+
+
```

Comment:
Perhaps you could look through your css files and delete any whitespace that's left over?